py/issue_triage/triage.py

```diff
@@ -790,5 +789,11 @@ def _add_triage_project(self, issue_info):
                             '|%(message)s|%(pathname)s|%(lineno)d|'),
                     datefmt='%Y-%m-%dT%H:%M:%S',
                     )
-
-  fire.Fire(IssueTriage)
+  
+  triager = IssueTriage()
```

Comment:
Can we continue to use Fire and have multiple entry points? Does GitHub actions not allow us to specify the command line to enter?

Reply:
Yes